src/ContaoCommunityAlliance/DcGeneral/Contao/Callback/ModelLabelCallbackListener.php

```diff
@@ -52,12 +55,41 @@ public function getArgs($event)
      *
      * @param ModelToLabelEvent $event The event being emitted.
      *
-     * @param string            $value The label text to use.
+     * @param string|array      $value The label text to use.
      *
      * @return void
      */
```

Comment:
I think it's not the information you need.

You should get the information from the ListingConfigInterface. there is a getShowColumn() getter.